files/rabbitmq-backup

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+DEST="$1"
+python /var/lib/rabbitmq/mnesia/*plugins-expand/rabbitmq_management-*/priv/www/cli/rabbitmqadmin export $DEST -c /etc/rabbitmqadmin.conf -N host_ssl -q
```

Comment:
Thanks.  I mostly prefer the new version because that's the documented place to get the script, whereas the path you used before seems to be an implementation detail we should not rely on.